src/main/stackpack/provisioning/templates/splunk-topology-instance-template.stj

```diff
@@ -7,14 +7,14 @@
         "autoExpireElements": false,
         "expireElementsAfter": 172800000,
         "id": -2,
-        "integrationType": "{{integrationType}}",
+        "integrationType": "splunk",
         "supportedDataTypes": [
           "TOPOLOGY_ELEMENTS"
         ],
         "supportedWindowingMethods": [],
         "topic": "{{topicName}}"
       },
-      "description": "DataSource that was generated for {{integrationType}} {{integrationUrl}}",
+      "description": "DataSource that was generated for {{instanceName}}",
```

Comment:
Do we want to add Splunk before {{ instanceName }} as well? similarly as we are doing for the Sync

Reply:
Added.